TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Mqs/DeleteMq/Index.cshtml.cs

```diff
@@ -51,10 +57,30 @@ public class IndexModel : PageModel
     [Display(Name = "More detail about the reason for deleting")]
     public string? DeletionReasonDetail { get; set; }
 
-    public void OnGet()
+    [BindProperty]
+    [Display(Name = "Upload evidence")]
+    [Required(ErrorMessage = "Select yes if you want to upload evidence")]
+    public bool? UploadEvidence { get; set; }
+
+    [BindProperty]
+    [FileExtensions(".bmp", ".csv", ".doc", ".docx", ".eml", ".jpeg", ".jpg", ".mbox", ".msg", ".ods", ".odt", ".pdf", ".png", ".tif", ".txt", ".xls", ".xlsx", ErrorMessage = "The selected file must be a BMP, CSV, DOC, DOCX, EML, JPEG, JPG, MBOX, MSG, ODS, ODT, PDF, PNG, TIF, TXT, XLS or XLSX")]
```

Comment:
Might be worth bundling this up into its own attribute deriving from `FileExtensions` - `EvidenceFile` perhaps? Feels like the kind of thing we'll be using a lot.

Reply:
Done